e2e/cephfs.go

```diff
@@ -345,6 +345,61 @@ var _ = Describe("cephfs", func() {
 				}
 			})
 
+			By("verify RWOP volume support", func() {
+				if !k8sVersionGreaterEquals(f.ClientSet, 1, 22) {
+					Skip("RWOP PVC access mode is only supported from v1.22+")
+				}
```

Comment:
```suggestion
				if k8sVersionGreaterEquals(f.ClientSet, 1, 22) {
					err=ValidateCephFSRWOP()
					if err!=nil{
					...
					}
				}
```
@humblec , this will lead to entire ci being skipped 
refer https://github.com/ceph/ceph-csi/pull/2680 

Example https://jenkins-ceph-csi.apps.ocp.ci.centos.org/blue/organizations/jenkins/mini-e2e-helm_k8s-1.20/detail/mini-e2e-helm_k8s-1.20/3467/pipeline/99

Reply:
ah. PR created before the referred one :) , will intend it properly.